### ISSUE TITLE
Version [1.8.0]: Sequence contains more than one element

### ISSUE THREAD

**Issue by rezabayesteh** [frankhommers/Hangfire.PostgreSql#176]
Hi,

I have noticed the following error in production when I updated to version 1.8.0 and seems to be related to commit 2e24835dd26878c842a51f28fb81a2d6bb3f5e47. Additionally, fetchJobSqlTemplate query needs to be revisited. I will try to create a pull request to fix this issue.

```
System.InvalidOperationException: Sequence contains more than one element
  Module "System.Linq.ThrowHelper", in ThrowMoreThanOneElementException
  Module "System.Linq.Enumerable", in SingleOrDefault
  Module "Hangfire.PostgreSql.PostgreSqlJobQueue+<>c__DisplayClass5_1", in Dequeue_Transaction { <lambda> }
  Module "Hangfire.PostgreSql.Utils.Utils", in TryExecute
  Module "Hangfire.PostgreSql.PostgreSqlJobQueue", in Dequeue_Transaction
  Module "Hangfire.PostgreSql.PostgreSqlJobQueue", in Dequeue
  Module "Hangfire.PostgreSql.PostgreSqlConnection", in FetchNextJob
  Module "Hangfire.Server.Worker", in Execute
  Module "Hangfire.Server.BackgroundProcessDispatcherBuilder", in ExecuteProcess
  Module "Hangfire.Processing.BackgroundExecution", in Run
```

**Comment by rezabayesteh** [comment]
This issue might be related to Postgresql bug:
https://www.postgresql.org/message-id/16497.1553640836%40sss.pgh.pa.us

So I would suggest to change the query to:
`
UPDATE """ + _options.SchemaName + @""".""jobqueue"" 
SET ""fetchedat"" = NOW() AT TIME ZONE 'UTC'
WHERE ""id"" = (
    SELECT ""id"" 
    FROM """ + _options.SchemaName + $@""".""jobqueue"" 
    WHERE ""queue"" = ANY (@queues)
    AND ""fetchedat"" {{0}}
    ORDER BY ""queue"", ""fetchedat"", ""jobid""
    FOR UPDATE SKIP LOCKED
    LIMIT 1
)
RETURNING ""id"" AS ""Id"", ""jobid"" AS ""JobId"", ""queue"" AS ""Queue"", ""fetchedat"" AS ""FetchedAt"";
";
`

**Comment by yongliu-mdsol** [comment]
@rezabayesteh good catch. can you create a PR for this?

**Comment by rezabayesteh** [comment]
@yongliu-mdsol Can you please review my PR #177 ?

**Comment by vytautask** [comment]
Should be fixed with 1.8.1 (published to Nuget just now)
